### DISCUSSION TITLE
绿联DPX4800部署日志显示问题

### DISCUSSION THREAD

**Discussion by mllt992** [natfrp/wiki#494] [使用问题] [answered]
设备：绿联DPX4800 标准版
镜像：natfrp/frpc 20260608 latest

![QQ截图20240608190429](https://github.com/natfrp/wiki/assets/59815753/42b86a63-9c7b-4e4d-beb7-b3c3e1b1dc1c)

尝试过的解决方案：
1. 不管添加还是删除环境变量LANG，值为 en_US.UTF-8，都无法解决该问题
2. 重启无法解决该问题

请问如何解决该问题，是否有其他方法查阅URL

[a comment by an automated account is not shown]

**Reply by mllt992** [reply]
感谢建议，您的方案是正确的。以下是我解决这个问题的操作步骤：
[https://www.cnblogs.com/mllt/p/18239029/nas-docker-nwct-natfrp](https://www.cnblogs.com/mllt/p/18239029/nas-docker-nwct-natfrp)

**Comment by fakeboboliu** [comment]
我们暂时没有借到还在使用原装系统的绿联 NAS 产品，
推荐您使用 Docker 版启动器的 web 管理，脱离终端需求从根本上解决问题。

**Reply by mllt992** [reply]
感谢建议，您的方案是正确的。以下是我解决这个问题的操作步骤：
https://www.cnblogs.com/mllt/p/18239029/nas-docker-nwct-natfrp

**Reply by fakeboboliu** [reply]
> 感谢建议，您的方案是正确的。以下是我解决这个问题的操作步骤： https://www.cnblogs.com/mllt/p/18239029/nas-docker-nwct-natfrp

感谢您的说明，请问我们能否在教程中引用您的文本和图片？
如果可以，我们会在引用后将您的 ID 添加到文档页最上面以表示感谢。

**Comment by mllt992** [comment]
完全没有问题 欢迎引用以帮助更多的人


https://www.cnblogs.com/mllt/p/18240219/nas_docker_gitea_https_nwct




【NAS】Docker Gitea+SakuraFrp+绿联DPX4800标 搭建私有代码托管平台



https://www.cnblogs.com/mllt/p/18239029/nas-docker-nwct-natfrp






【NAS】绿联NAS UGOS PRO 使用natfrp（Sakura Frp）内网穿透访问Docker应用




---原始邮件---
发件人: "bobo ***@***.***&gt;
发送时间: 2024年6月14日(周五) 晚上10:41
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;"State ***@***.***&gt;;
主题: Re: [natfrp/wiki] 绿联DPX4800部署日志显示问题 (Discussion #494)




  
感谢建议，您的方案是正确的。以下是我解决这个问题的操作步骤： https://www.cnblogs.com/mllt/p/18239029/nas-docker-nwct-natfrp
  
感谢您的说明，请问我们能否在教程中引用您的文本和图片？
 如果可以，我们会在引用后将您的 ID 添加到文档页最上面以表示感谢。
